pre-commit

```diff
@@ -17,7 +17,7 @@
 
 function use_bundle() {
   path_to_bundle=$(command -v bundle)
-  if [ -x "$path_to_bundle" -a -d ".bundle" ] ; then
+  if [[ -f Gemfile ]] && $path_to_bundle check 2>&1 >/dev/null; then
```

Comment:
Consider not using a bare variable here. `"${path_to_bundle}"`, as opposed to `$path_to_bundle`